### ISSUE TITLE
Crash when reloadData() meets the scroll 

### ISSUE THREAD

**Issue by grifas** [rechsteiner/Parchment#225]
I have 3 controllers. In each, at `viewDidAppear`, a request is sent and if there is a success,  calls `pagingViewController.reloadData()` to reload my custom PagingCells.

But I'm getting a crash when my second view controller appears, start the request and the user  returns quickly to the first view controller.  

Crash in `EMPageViewController.swift` at line 532.
```
if self.navigationDirection == .reverse { // check if direction changed
    self.didFinishScrolling(to: self.selectedViewController!)
    self.willScroll(from: self.selectedViewController, to: self.afterViewController!)
} 
```
 
The `self.selectedViewController` is equal to nil because the `reloadData()` calls `removeAllViewControllers()` :/

Is there a solution ? Is there another way to reload only one item from pagingViewController ?

**Comment by rechsteiner** [comment]
Thanks for the report! I will try to get this fixed as soon as possible.

**Comment by ChrisDunaetz** [comment]
@rechsteiner I made a PR to solve this issue: https://github.com/rechsteiner/Parchment/pull/230

**Comment by rechsteiner** [comment]
Released as part of v1.4.1 (#233) now ✌️
